### PR TITLE
chore(wren-ui): Enhance data source connection error & instructions

### PR DESCRIPTION
**Pull request by andreashimin**
## Description
Improving data source connection error messages helps users quickly understand and resolve issues.

- Provide `CONNECTION_ERROR` & `CONNECTION_REFUSED` error to client
- Refine UI in error alert component
- Provide instruction link when PG connection refused

## UI Screenshots
Connection Error
image
image

Connection Refused in PG
image

### CODE REVIEW

**Review by fredalai — APPROVED** [review]
LGTM for FE

File: wren-ui/src/apollo/server/connectors/postgresConnector.ts [hunk 1]
```diff
@@ -63,7 +64,15 @@ export class PostgresConnector
...
-      return false;
+      if (err.code === 'ECONNREFUSED') {
+        throw Errors.create(Errors.GeneralErrorCodes.CONNECTION_REFUSED, {
+          originalError: err,
+        });
+      } else {
+        throw Errors.create(Errors.GeneralErrorCodes.CONNECTION_ERROR, {
+          originalError: err,
+        });
+      }
```
Comment (wwwy3y3) [inline wren-ui/src/apollo/server/connectors/postgresConnector.ts:75]: maybe sth like following ?
```suggestion
     const errCode = (err.code === 'ECONNREFUSED') ? Errors.GeneralErrorCodes.CONNECTION_REFUSED : Errors.GeneralErrorCodes.CONNECTION_ERROR;
    throw Errors.create(errCode, {
        originalError: err,
      });
```

**Review by wwwy3y3 — COMMENTED** [review]
Backend, others lgtm

**Review by wwwy3y3 — APPROVED** [review]
LGTM

**Pull request merged**